ERCS/erc-7555.md

```diff
@@ -0,0 +1,211 @@
+---
+eip: 7555
+title: Single Sign-On for Account Discovery
+description: Discover accounts using a signing key that do not use the secp256k1 curve.
```

Comment:
it's really just any smart contract accounts right? (e.g. this could be used to connect to a safe without going through WalletConnect iiuc)

Reply:
Most likely yes, but leaving the door open for future innovation here.